lib/cartopy/crs.py

```diff
@@ -1702,8 +1786,21 @@ def __init__(self, central_longitude=0, globe=None):
                           'projection may be unreliable and should be used '
                           'with caution.')
 
+        if globe is None:
+            globe = Globe(semimajor_axis=WGS84_SEMIMAJOR_AXIS, ellipse=None)
```

Comment:
Perhaps we should add a factory... ``Globe.WGS84()``?

Reply:
Possibly, though with a different name, since this is WGS84 major radius on a sphere. It's really for backwards-compatibility since these projections simply ignore the semiminor axis.

Reply:
Good point.